exampleCourse/questions/demo/autograder/python/plots/question.html

```diff
@@ -11,10 +11,12 @@
     </p>
 
     <p>The setup code gives the following variables:</p>
-    <p><pl-external-grader-variables params-name="names_for_user"></pl-external-grader-variables></p>
+    <p><pl-external-grader-variables params-name="names_for_user" empty="true"></pl-external-grader-variables></p>
```

Comment:
Nit: can this be removed entirely?

Reply:
Will double check, but I don't think so. The Python grader requires certain parameters to be declared, even if they're just an empty list.

Reply:
Ah, sorry, should have been more specific. We can probably remove the `<p>` specifically, since I think that would render... nothing? an empty table? regardless, probably nothing useful. In all the other examples, the `<pl-external-grader-variables params-name="names_for_user" empty="true">` doesn't have any student-visible labels or anything.

Reply:
Removed here https://github.com/PrairieLearn/PrairieLearn/pull/7788/commits/db00a1677b7287220f6c6095f4bfc9b88d940fd3